Marlin/Conditionals_LCD.h

```diff
@@ -101,7 +110,8 @@
   #if ENABLED(ULTIMAKERCONTROLLER)              \
    || ENABLED(REPRAP_DISCOUNT_SMART_CONTROLLER) \
    || ENABLED(G3D_PANEL)                        \
-   || ENABLED(RIGIDBOT_PANEL)
+   || ENABLED(RIGIDBOT_PANEL)                   \
+   || ENABLED(REPRAPWORLD_KEYPAD)
```

Comment:
If it's true that not every REPRAPWORLD_KEYPAD is an ULTIPANEL then we shouldn't make that change just for the Anet, and @thinkyhead's suggestion makes sense to me.

In the context of this PR, the ANET_KEYPAD_LCD works with ULTIPANEL defined, so we still need to enable it, probably at line 49:

```
  #if ENABLED(ANET_KEYPAD_LCD)
    #define ULTIPANEL
    #define REPRAPWORLD_KEYPAD
   ...
```